### PR TITLE
DPR2-208 batch job

### PR DESCRIPTION
**Pull request by tom-ogle-moj**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -53,11 +53,9 @@ dependencies {
     compileOnly "org.apache.spark:spark-core_2.12:$sparkVersion"
     compileOnly "org.apache.spark:spark-sql_2.12:$sparkVersion"
     compileOnly "org.apache.hadoop:hadoop-aws:$hadoopVersion"
-    compileOnly "com.amazonaws:amazon-kinesis-client:$awsKinesisVersion" // transitively brings in java-sdk-core
     compileOnly "com.amazonaws:AWSGlueETL:$glueApiVersion"
 
-    implementation "com.amazonaws:aws-java-sdk-secretsmanager:$awsSdkVersion" // transitively brings in java-sdk-core
-    implementation "com.amazonaws:aws-java-sdk-glue:$awsSdkVersion" // transitively brings in java-sdk-core
+    implementation "com.amazonaws:aws-java-sdk-bundle:$awsSdkVersion"
```
Comment (tom-ogle-moj) [inline build.gradle:58]: Fixes the runtime issues when trying to run the app locally

File: src/main/java/uk/gov/justice/digital/job/udfs/JsonValidator.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package uk.gov.justice.digital.job.udf;
+package uk.gov.justice.digital.job.udfs;
```
Comment (tom-ogle-moj) [inline src/main/java/uk/gov/justice/digital/job/udfs/JsonValidator.java:1]: The package name clashes with spark udf

File: .circleci/config.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 version: 2.1
 
 orbs:
-  reporting: ministryofjustice/hmpps-reporting@1.0.25
+  reporting: ministryofjustice/hmpps-reporting@dev:0a8587270bfb07792227f9e64715852f0ee3eb0b
```
Comment (harichintala1) [inline .circleci/config.yml:4]: please use latest orb, ministryofjustice/hmpps-reporting@1.0.38

**Review by koladeadewuyi-moj — APPROVED** [review]

**Review by koladeadewuyi-moj — COMMENTED** [review]
Are the FIXME in the code intended for some future ticket?

**Comment by tom-ogle-moj** [comment]
@koladeadewuyi-moj I've brought up all the issues with @dominic-messenger-moj so that we can either raise Jira issues or add detail to existing issues. I'll also remove the FIXME and TODO comments I added from the code.

**Pull request merged**